OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -36,6 +36,9 @@ public class CargoInfo : ITraitInfo, Requires<IOccupySpaceInfo>
 		[Desc("When this actor is sold should all of its passengers be unloaded?")]
 		public readonly bool EjectOnSell = true;
 
+		[Desc("When this actor dies should all of its passengers be unloaded?")]
+		public readonly bool EjectOnDeath = false;
```

Comment:
It would be nice to have an option to eject the infantry with a health hit, but that makes this a bit more complicated so isn't required for my :+1: